packages/muelu/src/Transfers/Smoothed-Aggregation/MueLu_TentativePFactory_def.hpp

```diff
@@ -496,7 +504,9 @@ namespace MueLu {
 
             for (size_t k = 0, lnnz = 0; k < NSDim; k++) {
               // Skip zeros (there may be plenty of them, i.e., NSDim > 1 or boundary conditions)
-              const SC qr_jk = fineNS[k][aggToRowMapLO[aggStart[agg]+j]];
+              SC qr_jk = fineNS[k][aggToRowMapLO[aggStart[agg]+j]];
+              if(constantColSums) qr_jk = qr_jk / (double)aggSizes[agg];
+                fineNS[k][aggToRowMapLO[aggStart[agg]+j]];
```

Comment:
This line seems like it does a NOP ... I'm guessing it is a cut-and-paste leftover?

Reply:
Good catch.